volatility3/framework/layers/registry.py

```diff
@@ -96,6 +95,41 @@ def __init__(
                 f"Exception when setting hive {self.name} max address, using {hex(self._maxaddr)}",
             )
 
+    def _find_registry_process(self) -> Optional["extensions.EPROCESS"]:
+        """Walk the active process list and return the Registry process if it exists. Duplicates
+        PsList.list_processes() since pulling in the plugin causes problems.
+
+        Returns:
+            The Registry EPROCESS object if it exists, or None
+        """
+
+        kernel = self.context.modules.get("kernel")
```

Comment:
This shouldn't be hard coded as `kernel`, there's no guarantee what it'll be called.  So this should be `kernel_module_name`, although I'm just looking for if there's a nice way to get that from the information you've already got access to...